network-api/networkapi/wagtailpages/pagemodels/publications/article.py

```diff
@@ -113,6 +111,15 @@ class ArticlePage(FoundationMetadataPageMixin, Page):
         InlinePanel("footnotes", label="Footnotes"),
     ]
 
+    @property
+    def is_publication_article(self):
+        parents = self.get_ancestors()
+        parent_publication_pages = parents.filter(title__icontains="Publication")
+        if parent_publication_pages:
+            return True
+        else:
+            return False
```

Comment:
Yeah this is too fragile because eventually I'm sure a publication won't have "Publication" in the title. 

I think we just need to check if the parent page is an instance of a PublicationPage. Some pseudo-ish code:

```python
parent = self.get_parent() 
if isinstance(parent, PublicationPage):
    return True 
return False 
```

I don't think we need to check _all_ ancestors, just the parent page because Article Pages were initially supposed to only live under Publications, so the prev/next and breadcrumbs are already built for that based on the Parent always being an Article page.

Reply:
Thanks for the feedback Kalob! I appreciate it.

I have updated it to instead return whether or not the parents .__class__ is a "PublicationPage" model, similar to publication's 
[is_publication_page()](https://github.com/mozilla/foundation.mozilla.org/blob/76da7f8f5a06fa153a5b50bb0d21085c664bf494/network-api/networkapi/wagtailpages/pagemodels/publications/publication.py#L121) method.